blog/2023-03-10-conda-survey.md

```diff
@@ -0,0 +1,194 @@
+
+Dave Clements, on behalf of the Conda Communications Team
+
+---
+
+# How happy are people with conda?
+
+The conda survey asked 4 questions that we'll throw into the "happiness" bucket:
+
+[![Conda is my preferred package manager](/img/2023-03-10-conda-survey/conda-is-my-preferred-package-manager.png)](https://docs.google.com/spreadsheets/d/1rpWN6atSEsALvStPtAVIf_WMTxXXKaDKYpKRsvMpbOo/edit#gid=1699334366)
+[![I can access my desired packages](/img/2023-03-10-conda-survey/i-can-access-my-desired-packages.png)](https://docs.google.com/spreadsheets/d/1rpWN6atSEsALvStPtAVIf_WMTxXXKaDKYpKRsvMpbOo/edit#gid=1192657027)
+[![I can integrate conda with my workflow](/img/2023-03-10-conda-survey/i-can-integrate-conda-with-my-workflow.png)](https://docs.google.com/spreadsheets/d/1rpWN6atSEsALvStPtAVIf_WMTxXXKaDKYpKRsvMpbOo/edit#gid=1595912193)
+[![I am satisfied with conda](/img/2023-03-10-conda-survey/i-am-satisfied-with-conda.png)](https://docs.google.com/spreadsheets/d/1rpWN6atSEsALvStPtAVIf_WMTxXXKaDKYpKRsvMpbOo/edit#gid=1582523399)
+
+The news here is mostly good, but the results also highlight that things still need to be improved. For example, in "Conda is my preferred package manager" 61% had "happy" responses, but 25% were neutral, and almost 14% had "unhappy" responses.  That's 39% who can't say they are in the "happy" bucket.  For the other three questions the numbers outside the "happy" categories are 28%, 24%, and 32%.  Those are better, but still not where we want them.  See the specific feedback below for some reasons behind these numbers.
+
+# What needs improvement?
+
+The survey asked respondents what changes should be prioritized going forward.
+
+[![Desired conda changes](/img/2023-03-10-conda-survey/desired-conda-changes.png)](https://docs.google.com/spreadsheets/d/1rpWN6atSEsALvStPtAVIf_WMTxXXKaDKYpKRsvMpbOo/edit#gid=1229097249)
+
+## Speed
+
+The need for speed was the most prominent theme:
+
+* 71% prioritized speedups
+* 60% use mamba, which is known for being very fast compared to the standard conda solver.
+
+Some individual responses:
+
+* "just make libmamba the default solver already!!!!"
+* "Nearly 100% use Mamba nowadays due to speed, only fallback to Conda when something is going wrong"
+* "*Dear Goddess make the solver faster out of the box.*"
+
+This survey was taken in November and December 2022, which we feel was an inflection point in conda's performance profile.  The [22.11.0 conda release], which came out towards the end of the survey window, implemented parallel package download and extraction, and dropped the `experimental` tag from the `conda-libmamba-solver` (the solver was added in the March 2022 release). At the time of the survey, [only 19% of responsdents were using conda-libmamba-solver](https://docs.google.com/spreadsheets/d/1rpWN6atSEsALvStPtAVIf_WMTxXXKaDKYpKRsvMpbOo/edit#gid=1650032116).
```

Comment:
```suggestion
This survey was taken in November and December 2022, which we feel was an inflection point in conda's performance profile.  The [22.11.0 conda release], which came out towards the end of the survey window, implemented parallel package download and extraction, and dropped the `experimental` tag from the `conda-libmamba-solver` (the solver was added in the March 2022 release). At the time of the survey, [only 19% of respondents were using conda-libmamba-solver](https://docs.google.com/spreadsheets/d/1rpWN6atSEsALvStPtAVIf_WMTxXXKaDKYpKRsvMpbOo/edit#gid=1650032116).
```

Reply:
Are we missing a [22.11.0 conda release] link?

Reply:
Yes.  Will fix.